Content.Server/Connection/Whitelist/Conditions/ConditionManualWhitelist.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+using Content.Server.Database;
+using Robust.Shared.Network;
+
+namespace Content.Server.Connection.Whitelist.Conditions;
+
+public sealed partial class ConditionManualWhitelist : WhitelistCondition
+{
+    public override async Task<(bool, string)> Condition(NetUserData data)
+    {
+        var db = IoCManager.Resolve<IServerDbManager>();
```

Comment:
Do not pull in `IServerDbManager` via `IoCManager`. Move all the testing logic over to `ConnectionManager` to get rid of this OOP crap.